src/EPGImport/plugin.py

```diff
@@ -1065,19 +1080,20 @@ def Plugins(**kwargs):
 			wakeupfnc = getNextWakeup
 		),
 		PluginDescriptor(
-			name= _("EPGImport"),
+			name= "EPG importer",
```

Comment:
Shouldn't this be translatable?

Reply:
This is part of the execute in main menu code. It isn't used. Wording used is actually on line 731.

I will nonetheless submit the change to progress the matter.